src/app/child-dev-project/children/model/childSchoolRelation.ts

```diff
@@ -28,14 +28,17 @@ export class ChildSchoolRelation extends Entity {
     },
   })
   schoolId: string;
+
   @DatabaseField({ label: $localize`:Label for the class of a relation:Class` })
   schoolClass: string = "";
+
   @DatabaseField({
     dataType: "date-only",
     label: $localize`:Label for the start date of a relation:Start date`,
     description: $localize`:Description of the start date of a relation:The date a child joins a school`,
   })
   start: Date;
```

Comment:
I guess start and end date can be removed from here as well?

Reply:
Overwriting the description here to be a bit more user-friendly than the generic one (talking about joining a school).